scripts/gas-estimate.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * This script runs all tests with gas-reporter so you will see 
```

Comment:
super cool. can this give as an overview how much the snapshot token transfer would cost if there are 2, 5, 10, 100 pending votes/snapshots?